graphene/types/argument.py

```diff
@@ -102,9 +104,7 @@ def to_arguments(args, extra_args=None):
             raise ValueError(f'Unknown argument "{default_name}".')
 
         arg_name = default_name or arg.name
-        assert (
-            arg_name not in arguments
-        ), f'More than one Argument have same name "{arg_name}".'
+        assert arg_name not in arguments, f'More than one Argument have same name "{arg_name}".'
```

Comment:
I think this is causing the test to fail. It applies black formatting to the code an fails if the file changes. This is where it cites the changed file: https://travis-ci.org/github/graphql-python/graphene/jobs/753187127#L243